consensus/config/src/committee.rs

```diff
@@ -45,10 +45,10 @@ impl Committee {
             authorities.len()
         );
 
-        let total_stake = authorities.iter().map(|a| a.stake).sum();
+        let total_stake: u64 = authorities.iter().map(|a| a.stake).sum();
         assert_ne!(total_stake, 0, "Total stake cannot be zero!");
         let quorum_threshold = 2 * total_stake / 3 + 1;
-        let validity_threshold = (total_stake + 2) / 3;
+        let validity_threshold = total_stake.div_ceil(3);
```

Comment:
where do these changes come from?

Reply:
Locally clippy surfaces many more lints for me. Seems like maybe it doesn't pick up the `.clippy.toml` config ? I haven't figured out why yet, I'll look into it.
I've reverted this for now.